lib/DDG/Spice/Metar.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Metar;
+
+# Metar: Looks up the METAR information for a given airport. 
+# The input should be a 4 letter ICAO code.
+# It recognizes the query "metar ICAO" and "ICAO metar". 
+
+use DDG::Spice;
+
+spice to => 'http://avwx.rest/api/metar.php?station=$1&format=JSON';
+spice proxy_cache_valid => "418 1d"; # Do not cache responses.
+spice wrap_jsonp_callback => 1;
+
+triggers any => 'metar';
+
+handle words => sub {
```

Comment:
I think it would be easier to `handle remainder_lc` and  verify that the remainder matches the regular expression `[a-z]{4}`?

We could even go one step further and use https://metacpan.org/pod/Geo::ICAO or a similar library which can verify before we make the API call that the code is correct?

It would also be great if we could allow airport names to be passed in, but it's not necessary for V1 of this Instant Answer :+1: